src/pepr/operator/controllers/istio/injection.ts

```diff
@@ -45,7 +46,10 @@ export async function enableInjection(pkg: UDSPackage) {
       { force: true },
     );
 
-    await killPods(pkg.metadata.namespace, true);
+    // Kill the pods if we changed the value of the istio-injection label
+    if (originalInjectionLabel !== labels[injectionLabel]) {
```

Comment:
nit: Since we now have the const for it, using `originalInjectionLabel` on lines 26/30 would clean up/align all the places we reference original state.